Makefile.toml

```diff
@@ -113,7 +118,7 @@ script = '''
         envsubst < config/conffiles > ${PKG_BUILD_DIR}/control/conffiles
         chmod +x ${PKG_BUILD_DIR}/control/conffiles
 	for control_file in control preinst postinst prerm; do
-	    envsubst < ${PKG_SRC_DIR}/control/${control_file} > ${PKG_BUILD_DIR}/control/${control_file}
+	    envsubst "$(printf '${%s} ' $(env | sed 's/=.*//'))" < ${PKG_SRC_DIR}/control/${control_file} > ${PKG_BUILD_DIR}/control/${control_file}
```

Comment:
This change was made to support the use of bash variables in the control scripts. Without this change, `envsubst` will remove all local bash variables as it is trying to substitute  these local variables with non-existent environment variables. With this change, the local variables are retained if they are not substituted by `envsubst`